hs_sitemap/sitemaps.py

```diff
@@ -11,3 +13,17 @@ def items(self):
 
     def location(self, item):
         return reverse(item)
+
+
+class ResourcesSitemap(sitemaps.Sitemap):
```

Comment:
this was helpful for me:
https://docs.djangoproject.com/en/3.2/ref/contrib/sitemaps/#sitemap-class-reference